sdk/dotnet/Ad/GetApplication.cs

```diff
@@ -0,0 +1,102 @@
+// *** WARNING: this file was generated by the Pulumi Terraform Bridge (tfgen) Tool. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Azure.Ad
+{
+    public static partial class Invokes
+    {
+        /// <summary>
+        /// &gt; This content is derived from https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/d/azuread_application.html.markdown.
+        /// </summary>
+        public static Task<GetApplicationResult> GetApplication(GetApplicationArgs? args = null, InvokeOptions? options = null) => Pulumi.Deployment.Instance.InvokeAsync<GetApplicationResult>("azure:ad/getApplication:getApplication", args, options.WithVersion());
+    }
+
+    public sealed class GetApplicationArgs : Pulumi.ResourceArgs
+    {
+        /// <summary>
+        /// Specifies the name of the Application within Azure Active Directory.
+        /// </summary>
+        [Input("name")]
+        public Input<string>? Name { get; set; }
+
+        /// <summary>
+        /// Specifies the Object ID of the Application within Azure Active Directory.
+        /// </summary>
+        [Input("objectId")]
+        public Input<string>? ObjectId { get; set; }
+
+        public GetApplicationArgs()
+        {
+        }
+    }
+
+    [OutputType]
+    public sealed class GetApplicationResult
+    {
+        /// <summary>
+        /// the Application ID of the Azure Active Directory Application.
+        /// </summary>
+        public readonly string ApplicationId;
+        /// <summary>
+        /// Is this Azure AD Application available to other tenants?
+        /// </summary>
+        public readonly bool AvailableToOtherTenants;
+        public readonly string Homepage;
```

Comment:
lack of comment is consistent with aws.  wondering if we have a bug here, or if this really is just undocumented.